lib/ex_doc/autolink.ex

```diff
@@ -432,7 +413,7 @@ defmodule ExDoc.Autolink do
       {:type, _visibility} ->
         case config.language.try_builtin_type(name, arity, mode, config, original_text) do
           nil ->
-            if mode == :custom_link do
+            if mode == :custom_link or config.language == ExDoc.Language.Erlang do
```

Comment:
Is this check here to avoid warning in Elixir? If so, you can go ahead and warn. :)

Reply:
Yes it is, but if I remove it a lot (~20) of tests break (or not break, but they start to emit warnings to stdout).

Reply:
Ok, @wojtekmach please look into it afterwards then. :)

Ideally we want to avoid hardcoding the language here, an option would be better, but since this is temporary, it is fine by me. :)